.github/workflows/shellcheck.yaml

```diff
@@ -0,0 +1,32 @@
+name: ShellCheck
+
+on:
+  push:
```

Comment:
```suggestion
```
push will make it get run on `master`

Reply:
It works on master, it just doesn't work on a schedule: https://github.com/edwintorok/xen-api/actions/workflows/shellcheck.yaml
Anyway we can remove push.